src/test/resources/features/prisoner_search_with_aliases.feature

```diff
@@ -89,18 +89,6 @@ Feature: Prisoner Search results contain aliases
       | A1476AE    | 0             |           |
       | A1181MV    | 1             | O'VAUGHAN |
 
-  Scenario Outline: Search for prisoners with specified offender number using simple unprotected endpoint
-    Given a user has a token name of "GLOBAL_SEARCH"
-    When a search is made for prisoners with offender numbers of "<offenderNo>" using simple endpoint
```

Comment:
Removed tests associated to endpoint with new role - these already exist in the Resource Test